user/security-in-qubes/split-gpg.md

```diff
@@ -127,29 +127,32 @@ A note on `gpg` and `gpg2`:
 Throughout this guide, we refer to `gpg`, but note that Split-GPG uses `gpg2` under the hood for compatibility with programs like Enigmail (which now supports only `gpg2`).
 If you encounter trouble while trying to set up Split-GPG, make sure you're using `gpg2` for your configuration and testing, since keyring data may differ between the two installations.
 
-### Using Thunderbird + Enigmail with Split GPG ###
-
-However, when using Thunderbird with Enigmail extension it is not enough, because Thunderbird doesn't preserve the environment variables.
-Instead it is recommended to use a simple script provided by `/usr/bin/qubes-gpg-client-wrapper` file by pointing Enigmail to use this script instead of the standard GnuPG binary:
+### Advanced Configuration ###
 
-![tb-enigmail-split-gpg-settings-2.png](/attachment/wiki/SplitGpg/tb-enigmail-split-gpg-settings-2.png)
-
-The script also sets the `QUBES_GPG_DOMAIN` variable automatically based on the content of the file `/rw/config/gpg-split-domain`, which should be set to the name of the GPG backend VM.
-This file survives the AppVM reboot, of course.
+The `qubes-gpg-wrapper` script sets the `QUBES_GPG_DOMAIN` variable automatically based on the content of the file `/rw/config/gpg-split-domain`, which should be set to the name of the GPG backend VM. This file survives the AppVM reboot, of course.
 
     [user@work ~]$ sudo bash
     [root@work ~]$ echo "work-gpg" > /rw/config/gpg-split-domain
 
-#### Qubes 4.0 Specifics ####
-
-New qrexec policies in Qubes R4.0 by default require the user to enter the name of the domain containing GPG keys each time it is accessed.
-To improve usability for Thunderbird+Enigmail, in `dom0` place the following line at the top of the file `/etc/qubes-rpc/policy/qubes.Gpg`: 
+Split-GPG requires the user to enter the name of the AppVM containing GPG keys on each access within a certain time interval (300 seconds by default). To improve usability for applications like Thunderbird with Enigmail, in `dom0` place the following line at the top of the file `/etc/qubes-rpc/policy/qubes.Gpg`:
```

Comment:
This is not true. There are two confirmations:
 - generic qrexec confirmation - this asks you for the target name, and happens on _every_ call that has policy `ask`
 - split-gpg one - this asks only yes/no and allows the call for limited time (300s by default)

Policy set to `allow` prevents only the first one.

Reply:
Whoops, of course-- I mixed the two up while trying to reword it. Fixed.